1. What is the probability of rolling two six-sided dice and getting a value greater than 7?

In [3]:
c = 0
s = 0
for x in 1:6, y in 1:6
    if x + y > 7
        c += 1
    end
    s += 1
end
println(c)
println(s)

15
36


Or in one line:

In [11]:
sum([x + y > 7 for x in 1:6, y in 1:6])

15

If we say:  
H = values are greater than 7,  
So we have  
P(H) = 15 / 36

2. What is the probability of rolling three six-sided dice and getting a value greater than 7?

In [1]:
c = 0
s = 0
for x in 1:6, y in 1:6, z in 1:6
    if x + y + z> 7
        c += 1
    end
    s += 1
end
println(c)
println(s)

181
216


Or in one line:

In [4]:
sum([x + y + z > 7 for x in 1:6, y in 1:6, z in 1:6])

181

If we say:  
H = values are greater than 7,  
So we have  
P(H) = 181 / 216

3. The Yankees are playing the Red Sox. You’re a diehard Sox fan and bet your friend they’ll win the game. You’ll pay your friend $30 if the Sox lose and your friend will have to pay you only $5 if the Sox win. What is the probability you have intuitively assigned to the belief that the Red Sox will win?

H = Red Sox win  
O(H) = 30/5 = 6  
P(H) = 6/(1+6) = 6/7  